canadarm/launch/canadarm.launch.py

```diff
@@ -41,7 +43,7 @@ def generate_launch_description():
     )
 
     start_world = ExecuteProcess(
-        cmd=['ign gazebo', mars_world_model, '-r'],
+        cmd=['ign gazebo', leo_model, '-r'],
```

Comment:
Why is the world called "leo_model"? Not sure what this is referring to.

Reply:
Acronym for low earth orbit. It is on line 31.
`leo_model = os.path.join(FindPackageShare(package='canadarm').find('canadarm'), 'worlds/simple.world')`